travis_install_conda.sh

```diff
@@ -52,6 +52,7 @@ pip uninstall -y setuptools
 conda install setuptools
 
 conda update -q conda
+conda update numpy
```

Comment:
This doesn't belong here, we install the rest in a different CI script

Reply:
:'(

Reply:
https://github.com/pygae/clifford/blob/1b566721b877bed1854929642c1b80518e2a3165/.travis.yml#L14

Reply:
Just replace `numpy` with `"numpy >= 1.17"` there, I think. I don't know if spaces are allowed or not.